.github/workflows/main.yml

```diff
@@ -30,16 +30,15 @@ jobs:
         go-version: ${{ matrix.go-version }}
 
     - name: Run tests
-      run: |
-        # Run actual test as root as it uses CRIU.
-        sudo make test
-        # Test magicgen script
-        make -C scripts test
+      run: sudo env "PATH=$PATH:/usr/bin" make test
```

Comment:
As far as I know, `PATH` for the root user is set to
`/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin`
Are we sure we need to append it again?

Reply:
Thanks Prajwal, adding `/usr/bin` should not be necessary.